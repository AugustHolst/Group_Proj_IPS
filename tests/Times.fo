fun int main() = 
    let x = 4; y = 10  in 
    let z = x*y in 
    let xz = write(z) in


    let a = 168; b = 737363 in
    let testStor =  a * b in
    let ab1 = write(testStor) in

    let writeParentes = a * ~(b) in
    let ab2 = write(writeParentes) in

    let writeNegative = ~a*~y in
    let ab3 = write(writeNegative) in

    let writeNegativeandpositive= ~a*x in
    write(writeNegativeandpositive) 
    