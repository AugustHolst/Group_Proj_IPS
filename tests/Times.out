40 123876984 -123876984 1680 -672 